src/Domain/Creators/SphereTimeDependentMaps.hpp

```diff
@@ -131,15 +142,35 @@ struct TimeDependentMapOptions {
     std::optional<std::variant<KerrSchildFromBoyerLindquist>> initial_values{};
   };
 
-  using options = tmpl::list<InitialTime, ShapeMapOptions>;
+  struct TranslationMapOptions {
+    using type = TranslationMapOptions;
+    static std::string name() { return "TranslationMap"; }
+    static constexpr Options::String help = {
+        "Options for a time-dependent translation map in that keeps the outer "
+        "boundary fixed."};
+
+    struct InitialValues {
+      using type = std::array<std::array<double, 3>, 2>;
+      static constexpr Options::String help = {
+          "Initial values for the translation map."};
+    };
```

Comment:
I think we'll also need a second derivative here as well.

Reply:
Added